pkg/controller/registry/reconciler/grpc_test.go

```diff
@@ -70,6 +71,25 @@ func grpcCatalogSourceWithName(name string) *v1alpha1.CatalogSource {
 	return catsrc
 }
 
+func withPodDeletedButNotRemoved(objs []runtime.Object) []runtime.Object {
+	var out []runtime.Object
+	t := reflect.TypeOf(&corev1.Pod{})
+	for _, obj := range objs {
+		o := obj.DeepCopyObject()
+		if reflect.TypeOf(o) == t {
```

Comment:
Can the reflection library be replaced by a type assertion (which you already have), but that checks the type assertions bool return value.
```go
for _, obj := range objs {
    if pod, ok := obj.(*corev1.Pod); ok {
        // do stuff with pod
    }
}
```

Reply:
Done.